juju/python-libjuju
# [JUJU-3527] Added 3.1.2 and 3.2-beta2 schemas.

#### Description

Add missing facades for 3.1.2 and 3.2-beta schemas.

#### QA Steps

Bootstrap a 3.2-beta2 controller, connect to it, and check that all the facades are known.
